core/client.go

```diff
@@ -70,13 +71,25 @@ func (c *Client) Private(id *key.Identity) ([]byte, error) {
 	if err != nil {
 		return nil, err
 	}
-	resp, err := c.client.Private(id, &drand.PrivateRandRequest{obj})
+	resp, err := c.client.Private(id, &drand.PrivateRandRequest{Request: obj})
 	if err != nil {
 		return nil, err
 	}
 	return ecies.Decrypt(key.G2, ecies.DefaultHash, ephScalar, resp.GetResponse())
 }
 
+func (c *Client) DistKey(addr string) (kyber.Point, error) {
```

Comment:
Please add a secure boolean argument as the other methods of the Client.

Reply:
So we want to be able to get the distributed key without using TLS ? Isn't that risky ?